doc/classes/ProjectSettings.xml

```diff
@@ -2449,6 +2450,12 @@
 			If [code]true[/code], performs a previous depth pass before rendering 3D materials. This increases performance significantly in scenes with high overdraw, when complex materials and lighting are used. However, in scenes with few occluded surfaces, the depth prepass may reduce performance. If your game is viewed from a fixed angle that makes it easy to avoid overdraw (such as top-down or side-scrolling perspective), consider disabling the depth prepass to improve performance. This setting can be changed at run-time to optimize performance depending on the scene currently being viewed.
 			[b]Note:[/b] Depth prepass is only supported when using the Forward+ or Compatibility rendering method. When using the Mobile rendering method, there is no depth prepass performed.
 		</member>
+		<member name="rendering/driver/line_drawing/width" type="float" setter="" getter="" default="1.0">
+			Render hardware-based 3D lines using the specified value as a thickness (in pixels). Wire thickness does not depend on distance from the camera. This setting affects all lines drawn in 3D, including gizmos in the editor and debug draw options. This can be used to make debug drawing more visible, particularly when using a high-resolution viewport or on hiDPI displays.
+			[b]Note:[/b] This setting is only implemented in the Forward+ and Mobile rendering methods. In Compatibility, line drawing is always 1 pixel thick.
+			[b]Note:[/b] This setting relies on hardware capabilities for drawing lines thicker than 1 pixel. Support on Windows and Linux is generally good. Line widths greater than 1 pixel are rarely supported on Android. macOS and iOS do not support line widths greater than 1 pixel.
```

Comment:
Excessively repetitive. I find that steadily splitting the subjects into separate sentences is awesome, but not here...
```suggestion
			[b]Note:[/b] This setting requires the hardware to support drawing lines thicker than 1 pixel. This is usually supported on Windows and Linux, rarely supported on Android, and not supported on macOS and iOS.
```
Or similar.